data/common.yaml

```diff
@@ -14,3 +14,4 @@ profile_gpu::dcgm::install::dcgm_version: "2.4.6-1"  # 2.4.6-1 is known to work,
 profile_gpu::dcgm::telegraf::dcgm_telegraf_port: 8094
 profile_gpu::dcgm::telegraf::dcgm_telegraf_py_port: 5556
 profile_gpu::dcgm::telegraf::enable: true
+profile_gpu::dcgm::telegraf::nv_debug_check: "/var/spool/slurmd/nvperfenabled"
```

Comment:
Do we really want this to be a parameter in Hiera?

The reason I ask is because this path is hard coded in the custom fact. If that fact can't reference the hiera parameter for the file to check, it seems kind of pointless to be able to override the parameter via hiera.

Reply:
I've removed from Hiera.